### ISSUE TITLE
Useful if initialization scripts could be organised in sub directories

### ISSUE THREAD

**Issue by salk31** [gvenzl/oci-oracle-xe#147]
Hello,

First thanks for creating such great images, I lost a week trying to get the official ones to work.

The one tiny thing I miss from those is the ability to organise initialisation scripts in sub directories.

I've inherited a project that uses a lot of cross user/schema queries so I need to stuff three systems into one database. To keep the scripts sane and impose an order of execution it would be nice to store/order them in sub directories.

I'm not very good with shell scripts but I'd be up for attempting a pull request if helpful. Looking at the code it doesn't look too painful to make it recursive.

Cheers

Sam

**Comment by salk31** [comment]
The current order is fine for me, so order just based on name not type. I've got a working version where I just replaced "ignoring ${f}" with
`          if [ -d "${f}" ]; then
                      echo "CONTAINER: descending into ${f} ...";    run_custom_scripts ${f};    echo "CONTAINER: DONE: directory ${f}"
          else
                      echo "CONTAINER: ignoring ${f}"
          fi;`
That works but I thought it might be best to extract part of run_custom_scripts to limit the recursion to, so the messages made more sense and it didn't do the checks at the top each time.
I'm not that far off a pull request if that helps.

Sorry, I'm not sure I've answered your question directly.
